WootricSDK/WootricSDK/WTRiPADSocialShareView.m

```diff
@@ -147,7 +154,8 @@ - (void)setXConstraintsForFacebook:(BOOL)facebookAvailable twitter:(BOOL)twitter
       _twitterXConstraint.constant = -(totalWidth / 2) + socialButtonWidth;
```

Comment:
for all these methods (setXConstraintsForFacebook) you can have a Constraint class where you declare the positions as objects and just call them based on each scenario, other option is to move each combination to its own method and call them once the condition is met, this will help to readability.